### ISSUE TITLE
Math macros/guidelines

### ISSUE THREAD

**Issue by uschwes** [ethz-asl/asl-student-templates#13]
Do we have math macros and guidelines somewhere?

**Comment by pfankhauser** [comment]
I assume you talk about the LaTeX template for the report. There's some math definitions in the [ASL preamble](https://github.com/ethz-asl/asl-student-templates/blob/master/Report/preambles/asl_preamble.tex) but that's all. Feel free to add whatever is needed.

**Comment by uschwes** [comment]
Yes, I mean the latex reports. Do you know whether we have written down a styleguide for math notation somewhere? Like vectors small/bold and so on...
Do you think it would make sense to add macros for students so it's clear from the very beginning how we want equations to look like?

**Comment by pfankhauser** [comment]
The report follows the notation as defined in [isomath](http://www.ctan.org/pkg/isomath) ([examples](http://mirror.jmu.edu/pub/CTAN/macros/latex/contrib/isomath/isomath-test.pdf). So as long `\vec` and `\mat` are correctly used, there should be little confusion. We also given an example in the template under "2.6 Mathematische Formeln".

Do you think there's something that should be defined additionally? Feel free to add anything that might help.

**Comment by pfankhauser** [comment]
@uschwes do you think this is sufficiently covered? If so, I will close this issue.

**Comment by uschwes** [comment]
Should we put a link to the isomath example into our math section to stress that we want to follow that convention? 
It might be overlooked easily.

**Comment by pfankhauser** [comment]
Sorry for not following up, feel free to add a link or any more information.

**Comment by uschwes** [comment]
Haha, me too :-) 
I hope I can make a proposal at some point how to help students avoid common mistakes. 
This might include a section on negative examples and the usage of e.g. siunitx, acro,...
Maybe we can converge on some minimum set of guidelines.
